src/herder/HerderSCPDriver.cpp

```diff
@@ -619,6 +624,11 @@ HerderSCPDriver::combineCandidates(uint64_t slotIndex,
     CLOG(DEBUG, "Herder") << "Combining " << candidates.size() << " candidates";
     mSCPMetrics.mCombinedCandidates.Mark(candidates.size());
 
+    if (candidates.empty())
```

Comment:
Why is this needed now, when it wasn't before?

Reply:
It's not needed; it's an assertion, which I thought, if a caller of `combinedCandidates()` were ever to turn out to have a bug that called that method with an empty `candidates` list, would make the source clearer than the `NULL` pointer dereference that we'd get without it.  I'll remove it.